### DISCUSSION TITLE
Lower Accuracy with Dynamic Models as compared to the Static architecture.

### DISCUSSION THREAD

**Discussion by AdityaJainTD** [ContinualAI/avalanche#1077] [Q&A] [answered]
Hi,

I have been experimenting with Dynamic Models provided by Avalanche. I am using nc_benchmark as the setting and have been using plugins such as Naive, Replay, EWC, Hybrid etc. And since I am using these strategies/plugins with my understanding I don't have to explicitly call 'models.adaptation' as the plugins will implicitly call them to update the parameters. So when I compare the accuracy with respect to static models (with fixed output nodes), the accuracy is quite bad. Is this normal? or is there something which is missing in my training loop?

**Comment by AdityaJainTD** [comment, marked as answer]
Update:- There was a minor bug in my codebase. The accuracy is comparable to that of static models. Apologizes for any confusion caused.

**Reply by ZaidRZ** [reply]
Dear AdityaJainTD
can i contact with you?
